themes/src/main/resources/theme/base/login/resources/js/webauthnRegister.js

```diff
@@ -0,0 +1,130 @@
+import { base64url } from "../../../common/keycloak/node_modules/rfc4648/lib/rfc4648.js";
+
+window.registerByWebAuthn = (challenge, userid, username, signatureAlgorithms, rpEntityName, rpId, attestationConveyancePreference, authenticatorAttachment, requireResidentKey, userVerificationRequirement, createTimeout, excludeCredentialIds, errmsg) => {
```

Comment:
This should not be leaked to the window.

Reply:
I fixed the codes as you suggested.